docs/readthedocs/source/doc/Nano/Overview/pytorch_inference.md

```diff
@@ -41,6 +47,8 @@ Before you go ahead with these APIs, you have to make sure BigDL-Nano is correct
     - `ONNXRuntime <https://onnxruntime.ai/>`_: ``pip install onnx onnxruntime onnxruntime-extensions onnxsim neural-compressor``
 
     We recommand installing all dependencies by ``pip install --pre --upgrade bigdl-nano[pytorch,inference]``, because you may run into version issues if you install dependencies manually.
+
+    For the case you do not need the optimizations from INC, OpenVINO, or ONNXRuntime (e.g. you only want to use multi-instance inference, optimization from IPEX (Intel Extension for PyTorch) or TorchScript, etc.), you could just install BigDL-Nano for PyTorch through ``pip install --pre --upgrade bigdl-nano[pytorch]``.
```

Comment:
From "We recommand ins..." to "--upgrade bigdl-nano[pytorch]``." is quite the same thing as the previous part of this note. Should we just delete them?